docker/run.sh

```diff
@@ -1,12 +1,18 @@
 #!/bin/sh
+#
+# Run the docker container.
+
 . docker/env.sh
 docker stop $CONTAINER_NAME
 docker run \
   -dit \
   --gpus all \
   -v $PWD:/workspace \
-  -p 6008:6008 \
+  -p $HOST_PORT:$CONTAINER_PORT \
```

Comment:
For a while, it's no problem to use HOST_PORT. However, we (or users) will use other ports in the future. So change HOST_PORT, CONTAINER_PORT to MLFLOW_HOST_PORT and MLFLOW_CONTAINER_PORT, respectively.

Reply:
I understand. I'll change it.